### ISSUE TITLE
Add xr to Deken

### ISSUE THREAD

**Issue by Yzrsah** [defaultxr/xr-abstractions#2]
This is a nice library. It would be nice if you added it to Deken so people can easily install it in I.e. PlugData which has a Deken package manager.

**Comment by defaultxr** [comment]
Hi, and thanks! I agree it would be nice to have this library added to Deken. The main reason I haven't pursued that yet is because there is a lot of cleanup that needs to be done before I feel that this library is worth publishing officially as a "version 1.0". I wouldn't want to publish this to Deken and have people depend on something I end up changing or removing.

I don't use Pd as often as I used to but I've been gradually going through the library and cleaning it up when I do open up Pd. I'll keep in mind that there is still interest in these abstractions and I'll try to find more time to work on them as I'm able.
